doc/Learn/Tutorials/Http/HowTo-Refit.md

```diff
@@ -0,0 +1,219 @@
+            {
+                hostBuilder.UseHttp();
+            });
+    ...
+    ```
+
+* This feature requires the `Uno.Extensions.Http` package. It uses [Microsoft Extensions](https://www.nuget.org/packages/Microsoft.Extensions.Http) for any HTTP-related [work](https://learn.microsoft.com/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests#benefits-of-using-ihttpclientfactory) such as naming or configuring the `HttpClient` instance associated with your endpoints.
+
+### 2. Define the live interface
+
+* Unlike standard HTTP endpoints, Refit endpoints you define will be registered as a service with the `AddRefitClient<T>()` extension method, where the type argument `T` corresponds to an interface you define.
+
+* This interface will be used to generate the strongly-typed client. It will be used to make requests to the web service and deserialize responses.
+
+* For the purposes of this tutorial, we will create and register an interface for the Chuck Norris facts web API 
+
+* Start by defining an interface `IChuckNorrisEndpoint` with a method `Search()` that returns a `Task` of type `ApiResponse<ChuckNorrisData>`:
+    ```csharp
+    [Headers("Content-Type: application/json")]
+    public interface IChuckNorrisEndpoint
+    {
+        [Get("/jokes/search")]
+        Task<ApiResponse<ChuckNorrisData>> Search(CancellationToken ct, [AliasAs("query")] string searchTerm);
+    }
+    ```
+
+    > [!NOTE]
+    > A class named `ChuckNorrisData` will be defined in the subsequent sections of this guide. It will be used to deserialize the response from the API. 
+
+* The `Headers` attribute is used to specify the `Content-Type` header for the request. `Get` specifies the relative path for a `Search` request. Notice that the `searchTerm` parameter for the request is aliased as `query` using an `AliasAs` attribute.
+
+### 3. Register the endpoint
+
+* The `AddRefitClient<T>()` extension method is used to register the endpoint with the service collection.
+
+* This extension method can take a delegate as its argument, but the recommended way to configure a HTTP client is to specify a configuration section name. This allows you to configure the added HTTP client using the `appsettings.json` file. 
+
+* Add the Refit client to the service collection with the following code:
+
+    ```csharp
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var appBuilder = this.CreateBuilder(args)
+            .Configure(hostBuilder =>
+            {
+                hostBuilder.UseHttp((context, services) =>
+                    services.AddRefitClient<IChuckNorrisEndpoint>(context)
+                );
+            });
+    ...
+    ```
+
+* Define a configuration section in the `appsettings.json` file. While the default behavior is to use the platform-native HTTP handler, this can be configured. 
+
+    ```json
+    {
+        "ChuckNorrisEndpoint": {
```

Comment:
Is it worth explaining how this name is auto-magically mapped to the interface name? - at least, I assume how this works as it isn't explained.

Reply:
Yes, point out that by default the name of the section is the interface name without the leading "I". 
Also point out that this can be overridden by specifying a name when registering the interface